base/reflection.jl

```diff
@@ -1127,20 +1127,23 @@ function code_lowered(@nospecialize(f), @nospecialize(t=Tuple); generated::Bool=
     end
     world = get_world_counter()
     world == typemax(UInt) && error("code reflection cannot be used from generated functions")
-    return map(method_instances(f, t, world)) do m
+    ret = CodeInfo[]
+    for m in method_instances(f, t, world)
         if generated && hasgenerator(m)
             if may_invoke_generator(m)
-                return ccall(:jl_code_for_staged, Any, (Any, UInt), m, world)::CodeInfo
+                push!(ret, ccall(:jl_code_for_staged, Any, (Any, UInt), m, world)::CodeInfo)
```

Comment:
Doesn't this change the behavior of the function, by adding two methods to `ret` in the current iteration? Perhaps it should be like this to match current behavior:
```suggestion
                push!(ret, ccall(:jl_code_for_staged, Any, (Any, UInt), m, world)::CodeInfo)
                continue
```
Or is this change intentional? But then the PR description should make that clear, no?

Reply:
Thanks, that's my mistake.